apps/examples/wifi_manager_sample/wm_test_usage.h

```diff
@@ -75,6 +75,19 @@
 	"	 wm_test power [mode]\n"				\
 	"	 mode examples : on, off\n\n"
 
+#define WT_INTEROP_USAGE						\
+	"\n Interoperability test: \n"				\
+	"	wm_test interop \n"
+
+#define WT_ADD_AP_USAGE					\
+	"\n Add AP config in file wm_aplist.conf \n"				\
+	"	wm_test add_ap [ssid] [security mode] [password]\n"		\
+
+#define WT_DISPLAY_AP_USAGE					\
+	"\n Display AP config from file wifi_interop.conf: \n"				\
```

Comment:
Please change the file name from wifi_interop.conf to wm_aplist.conf here.

Reply:
resolved